spec/namespaces/indices.yaml

```diff
@@ -938,6 +938,24 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/indices.delete@200'
+  /{index}/_tier/warm:
+    post:
+      operationId: indices.tier_warm.0
+      x-operation-group: indices.tier
+      x-version-added: '1.0'
```

Comment:
This should be the actual version when this is added.

Reply:
since this will be behind a feature flag for now, we havent released it yet - should this pr not be merged until its released?

Reply:
@dblock what do u suggest here?

Reply:
I think we need a bit of work to add support for experimental features, I opened https://github.com/opensearch-project/opensearch-api-specification/issues/400. Locally you can boot a next version of OpenSearch with this feature flag enabled to get a PR finished for now.